src/app/components/content-action-bar/ContentActionBar.jsx

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+const buttonStyle = {
+    STANDARD: "standard",
+    POSITIVE: "positive",
+    PRIMARY: "primary",
+    WARNING: "warning",
+    SUBTLE: "subtle",
+    INVERT_PRIMARY: "invert-primary"
+};
+const buttonStyleArray = Object.values(buttonStyle);
+const propTypes = {
```

Comment:
can we start using propTypes like recommended on the react docs https://reactjs.org/docs/typechecking-with-proptypes.html.
```
ContentActionBar.propTypes = {
  .... 
};
```